packages/checkbox/theme/lumo/vaadin-checkbox-styles.js

```diff
@@ -22,6 +22,7 @@ registerStyles(
       user-select: none;
       cursor: default;
       outline: none;
+      --_checkbox-size: calc(var(--lumo-size-m) / 2);
```

Comment:
We could define this as `--_checkbox-size: var(--vaadin-checkbox-size, calc(var(--lumo-size-m) / 2));` and use it internally as such to avoid having to repeat `var(--vaadin-checkbox-size, var(--_checkbox-size));`

Reply:
Thanks, done.